application/api/checkconnection.py

```diff
@@ -59,6 +59,13 @@ def post(self, session):
                                 message=APIMessages.DB_TYPE_NAME,
                                 http_status_code=STATUS_BAD_REQUEST,
                                 data={})
+        # Checking spaces in username and hostname
+        spacecount_dbusername = db_data["db_username"].find(" ")
+        spacecount_dbhostanme = db_data["db_hostname"].find(" ")
```

Comment:
Can we use as follows:

" " in db_data["db_username"] or " " in db_data["db_hostname"]

Reply:
ya we can use